### ISSUE TITLE
confused with paper and its corresponding code.

### ISSUE THREAD

**Issue by sycophant-stone** [BoLiu-SVCL/meta-open#3]
what i get from the paper is 
(1) there is a stage-like train procedure
(2) there is a _h_ like meta-model to **aux** the backbone.
(3) there seems a  **unseen classes** in the test.

what i get from the code is
(1) the loss-function could consists three parts.
        i. the few-shot loss-functions, aka, the -Euclidean distance.
       ii. the open-set loss-functions, aka, the papers' open-set evaluation algorithm.
      iii. the aux part. this part use the _fc_ to evaluation logits. **is the fc the paper's h-thing?**
(2) the dataloader gather the same-class for `few-shot` ,`open-set` and `aux` types, but different images from the train-set. could it be possible to the unseen classes?

thanks a lot.

**Comment by wtmlon** [comment]
I have similar confusion. I want to know what is  **h** that passed from Meta-learner to learner in the paper?
